src/viur/core/modules/moduleconf.py

```diff
@@ -1,12 +1,14 @@
 import logging
 from viur.core import Module, conf, db, current, i18n, tasks, skeleton
 from viur.core.bones import StringBone, TextBone, SelectBone, TreeLeafBone
-from viur.core.bones.text import _defaultTags
 from viur.core.prototypes import List
 
 
 MODULECONF_KINDNAME = "viur-module-conf"
 
+_LIMITED_HTML = conf.bone_html_default_allow.copy() \
```

Comment:
There is no need to create a copy before you create a new dict anyway

Reply:
you're right, this was a adapted from the previous implementation.
```suggestion
_LIMITED_HTML = conf.bone_html_default_allow \
```